pkg/apis/targets/v1alpha1/googlesheet_lifecycle.go

```diff
@@ -103,8 +103,8 @@ func (s *GoogleSheetTargetStatus) PropagateAvailability(ksvc *servingv1.Service)
 }
 
 // MarkNoKService sets the condition that the service is not ready
-func (a *GoogleSheetTargetStatus) MarkNoService(reason, messageFormat string, messageA ...interface{}) {
-	AwsCondSet.Manage(a).MarkFalse(ConditionServiceReady, reason, messageFormat, messageA...)
+func (s *GoogleSheetTargetStatus) MarkNoService(reason, messageFormat string, messageA ...interface{}) {
+	AwsCondSet.Manage(s).MarkFalse(ConditionServiceReady, reason, messageFormat, messageA...)
```

Comment:
oh man.. lol just noticed this xd

Reply:
https://github.com/triggermesh/triggermesh/issues/74

Reply:
Uh, oh. AWS in a Google target 😄